Source/Scene/Billboard.js

```diff
@@ -152,6 +152,13 @@ define([
         this._mode = SceneMode.SCENE3D;
 
         this._updateClamping();
+
+        var scene = this._billboardCollection._scene;
+        if (defined(scene)) {
+            scene.terrainProviderChanged.addEventListener(function() {
```

Comment:
Would it be better to do this once for the billboard collection, then loop over all the billboards in the collection?  Save O(n) memory?